code/game/gamemodes/heist/heist.dm

```diff
@@ -57,187 +55,186 @@ var/global/list/obj/cortical_stacks = list() //Stacks for 'leave nobody behind'
 	return 1
 
 /datum/game_mode/heist/pre_setup()
+	var/index = 1
+	for(var/datum/mind/raider in raiders)
+		if(index > raider_spawn.len)
+			index = 1
```

Comment:
rather than this complex loop you could just do
```
var/index = 1
for(var/datum/mind/raider in raiders)
    raider.current.loc = raider_spawn[index % (raider_spawn.len + 1)]
    make_raider_gear(raider_gear[index % (raider_spawn.len + 1)])
    index++
```